inorbit_edge/tests/demo/example.py

```diff
@@ -158,11 +163,21 @@ def my_command_handler(robot_id, command_name, args, options):
     assert inorbit_api_key, "Environment variable INORBIT_API_KEY not specified"
 
     # Create robot session factory and session pool
-    robot_session_factory = RobotSessionFactory(
-        endpoint=inorbit_api_endpoint,
-        api_key=inorbit_api_key,
-        use_ssl=False if inorbit_api_use_ssl == "false" else True,
-    )
+    # If a robot_key is specified, use it as for authentication. Otherwise, use
+    # the api_key.
+    if inorbit_robot_key:
+        robot_session_factory = RobotSessionFactory(
+            endpoint=inorbit_api_endpoint,
+            robot_key=inorbit_robot_key,
```

Comment:
I'd add a big NOTE on this section describing the Factory limitation with a single `robotKey`.